utils/utils.py

```diff
@@ -630,26 +627,6 @@ def compare_config_yamls(yaml1: dict, yaml2: dict, update_yaml1: bool = False) -
                     log.info(f'Value in yaml1 updated')
 
 
-def load_obj(obj_path: str, default_obj_path: str = '') -> any:
-    """
-    Extract an object from a given path.
-
-    :param obj_path: (str) Path to an object to be extracted, including the object name.
-    :param default_obj_path: (str) Default path object.
-
-    :return: Extract object. Can be a function or a class or ...
-
-    :raise AttributeError: When the object does not have the given named attribute.
-    """
-    obj_path_list = obj_path.rsplit('.', 1)
-    obj_path = obj_path_list.pop(0) if len(obj_path_list) > 1 else default_obj_path
-    obj_name = obj_path_list[0]
-    module_obj = imp.import_module(obj_path)
-    if not hasattr(module_obj, obj_name):
-        raise AttributeError(f"Object `{obj_name}` cannot be loaded from from `{obj_path}`.")
-    return getattr(module_obj, obj_name)
-
-
```

Comment:
@CharlesAuthier same here. Let me know if you had anytning in my mind for further developments with this function. If the purpose was just to use in GDL.py to reach the right method, I think hydra.utils.get_method() does the job.